fixtures/Terms/test.md

```diff
@@ -3,3 +3,7 @@
 Look at the bottom-left of your screen.
```

Comment:
https://ibmdocs-test.dcs.ibm.com/docs/en/ibm-style?topic=elements-ui#location-of-interface-elements

We're supposed to avoid directions if possible, so something like bottom-left should be flagged as a warning.

Reply:
Good spot, that sentence is actually a test designed to check the warning triggers correctly; however, I've noticed the suggestions it gives don't match the writing guide you linked. I've made an issue https://github.com/IBM/ibm-quantum-style-guide/issues/11.